apigrpc/apigrpc.swagger.json

```diff
@@ -4713,6 +4713,10 @@
         "environment": {
           "$ref": "#/definitions/ValidatedPurchaseEnvironment",
           "description": "Whether the purchase was done in production or sandbox environment."
+        },
+        "seenBefore": {
+          "type": "boolean",
+          "description": "Whether the purchase had already been validated by Nakama before."
```

Comment:
We should keep this, but can we revert changes to `apigrpc/apigrpc.pb.go` and `apigrpc/apigrpc.proto`? Looks like superfluous changes.